config/initializers/rswag_ui.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Rswag::Ui.configure do |c|
+  # List the Swagger endpoints that you want to be documented through the
+  # swagger-ui. The first parameter is the path (absolute or relative to the UI
+  # host) to the corresponding endpoint and the second is a title that will be
+  # displayed in the document selector.
+  # NOTE: If you're using rspec-api to expose Swagger files
+  # (under swagger_root) as JSON or YAML endpoints, then the list below should
+  # correspond to the relative paths for those endpoints.
+
+  c.swagger_endpoint "/api-docs/openapi#{ENV.fetch('TEST_ENV_NUMBER', '')}.yaml", 'API V1 Docs'
```

Comment:
```suggestion
  c.swagger_endpoint "/api-docs/openapi.yaml", 'API V1 Docs'
```

I don't think we need to care about test parallelization here

Reply:
@santib sure, I was use it because the file are defined with this value